### PR TITLE
Dockerfile not found error handling

### LINKED ISSUE #9520: `az acr task create` doesn't enforce required -f when supplying --context
## Describe the bug
Az CLI throws exception instead of printing error when supplying a context without specifying `-f`. If I specify `-f Dockerfile` then it doesn't throw exception

**Command Name**
`az acr task create`

**Errors:**
```
'NoneType' object has no attribute 'endswith'
Traceback (most recent call last):
python3.7/site-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 560, in execute
    raise ex
cli/core/commands/__init__.py, ln 618, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
cli/core/commands/__init__.py, ln 611, in _run_job
    six.reraise(*sys.exc_info())
...
cli/core/commands/__init__.py, ln 297, in __call__
    return self.handler(*args, **kwargs)
azure/cli/core/__init__.py, ln 453, in default_command_handler
    return op(**command_args)
cli/command_modules/acr/task.py, ln 84, in acr_task_create
    if file.endswith(ALLOWED_TASK_FILE_TYPES):
AttributeError: 'NoneType' object has no attribute 'endswith'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az acr task create -n test-foo -r jcdemoregistry --context https://github.com/jamesc/habitat --commit-trigger-enabled false --pull-request-trigger-enabled false -t habitat:{{.Run.ID}}`

## Expected Behavior
Print error message saying that `-f <FILE>` is required.

## Environment Summary
```
Darwin-18.6.0-x86_64-i386-64bit
Python 3.7.3
Shell: bash

azure-cli 2.0.64 *

Extensions:
alias 0.5.2
interactive 0.4.2
db-up 0.1.8

```

### PR DESCRIPTION
**Pull request by ElianaLiberman**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

FIXES #9520

### CODE REVIEW

**Comment by northtyphoon** [comment]
https://travis-ci.org/Azure/azure-cli/jobs/545512074
pylint:
************* Module azure.cli.command_modules.acr.task
R: 35, 0: Too many statements (51/50) (too-many-statements)

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py [hunk 1]
```diff
@@ -83,27 +83,30 @@ def acr_task_create(cmd,  # pylint: disable=too-many-locals
...
-                image_names=image_names,
-                is_push_enabled=not no_push,
-                no_cache=no_cache,
-                docker_file_path=file,
-                arguments=(arg if arg else []) + (secret_arg if secret_arg else []),
-                context_path=context_path,
-                context_access_token=git_access_token,
-                target=target
-            )
+            raise CLIError("--file <Dockerfile> not found")
```
Comment (northtyphoon) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py]: --file can be Dockerfile or tasks yaml file. You can say "missing --file/-f argument"

Comment (ElianaLiberman) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py]: Done

**Comment by ElianaLiberman** [comment]
> https://travis-ci.org/Azure/azure-cli/jobs/545512074
> pylint:
> ************* Module azure.cli.command_modules.acr.task
> R: 35, 0: Too many statements (51/50) (too-many-statements)

**Pull request merged**